backend/agents/create_tasks_agent.py

```diff
@@ -0,0 +1,41 @@
+from utils import Config
+from utils import call_model
+import logging
+
+logger = logging.getLogger(__name__)
+config = Config()
+
+create_tasks_prompt = """
+    You are an agent who specialises in breaking down questions into smaller tasks that are performed to complete
+    the question.
+
+    You are dedicated to performing the job of breaking down a user question into up to 5 tasks that are easy
+    to understand and manageable in size.
+
+    A user will provide a goal that they want to perform and you will create tasks that when all of them are performed,
+    will provide an answer for the user question.
+
+    To determine the tasks you will create, you will use reasoning, any context you have and will attempt to criticise
+    each task in order to find the best way possible for tasks to answer the overall question.
+
+    If you are unable to breakdown the question into tasks, say that unfortunately you are unable to break down
+    the question.
+
+    You are connected to a neo4j database, as a source of information. You do not need to create a task to connect
+    or retrieve user data from the database.
+
+    EG. prompt: "Which bank should I choose to open a savings account based on the best interest rate
+    between Lloyds Bank and Tesco Bank?"
+    1) Find the interest rate for Lloyds Bank's savings account. Explanation: {your reasoning}
+    2) Find the interest rate for Tesco Bank's savings account. Explanation: {your reasoning}
+    3) Compare both values.
+
+    EG. prompt: "How much did I spend last month on my debit card?"
+    1) Find all spending transactions last month on the user's debit card. Explanation: {your reasoning}
+    2) Sum all transactions. Explanation: {your reasoning}
+"""
+
+def create_tasks(user_prompt):
+    logger.info("create_tasks function is called")
+    response = call_model(create_tasks_prompt, user_prompt)
+    return(response)
```

Comment:
is there a reason why we're using a tuple as the return type here? Reading the method name `create_tasks` makes me think some sort of array / list will be returned